Typescript 2.9 generated declaration types have wrong import path

With formik beta-3, got an error like this one : 
```
/frontend/node_modules/formik/dist/connect.d.ts
(3,59): Cannot find module '../node_modules/create-react-context/lib/index'.

/frontend/node_modules/formik/dist/connect.d.ts
(3,189): Cannot find module '../node_modules/create-react-context/lib/index'.
```
- Formik Version:1.0.0-beta.3
- React Version:16.4.1
- TypeScript Version:2.9.2
- OS: windows
- Node Version:10.5.0

```
/projects/demo/node_modules/formik/dist/connect.d.ts
(3,59): Cannot find module '../node_modules/create-react-context/lib/index'.
```
Same issue here with beta 4. Deleted the node_modules as suggested and did a reinstall.

Switching back to 0.11.11 solved it.

--- a/package.json
+++ b/package.json
@@ -43,7 +43,7 @@
   "dependencies": {
     "create-react-context": "^0.2.2",
     "deepmerge": "^2.1.1",
-    "hoist-non-react-statics": "^2.5.0",
+    "hoist-non-react-statics": "^2.5.5",
     "lodash.clonedeep": "^4.5.0",
     "lodash.topath": "4.5.2",
     "prop-types": "^15.6.1",
@@ -88,7 +88,7 @@
     "tsc-watch": "^1.0.21",
     "tslint": "5.5.0",
     "tslint-react": "3.2.0",
-    "typescript": "^2.9.2",
+    "typescript": "^3.0.0-dev.20180711",
     "yup": "0.21.3"
   },
   "lint-staged": {

